resource/closecaption_german.txt

```diff
@@ -800,45 +800,45 @@
 "[english]crash.NoAmmo0"		"<clr:238,180,34>Crash: Out of ammo!"
 "crash.Medic0"		"<clr:238,180,34>Crash: Sanitäter!"
 "[english]crash.Medic0"		"<clr:238,180,34>Crash: Medic!"
-"crash.Reloading0"		"<clr:238,180,34>Crash: Sichern und laden!"
+"crash.Reloading0"		"<clr:238,180,34>Crash: Durchladen!"
 "[english]crash.Reloading0"		"<clr:238,180,34>Crash: Lock and load!"
-"crash.Reloading1"		"<clr:238,180,34>Crash: Nachladen, gib mir Deckung!"
+"crash.Reloading1"		"<clr:238,180,34>Crash: Nachladen, gebt mir Deckung!"
 "[english]crash.Reloading1"		"<clr:238,180,34>Crash: Reload, cover me!"
 "crash.Reloading2"		"<clr:238,180,34>Crash: Nachladen!"
 "[english]crash.Reloading2"		"<clr:238,180,34>Crash: Reload!"
 "crash.Reloading3"		"<clr:238,180,34>Crash: Nachladen!"
 "[english]crash.Reloading3"		"<clr:238,180,34>Crash: Reload!"
-"crash.SentryInPlace0"		"<clr:238,180,34>Crash: Sentry Gun aktiv!"
+"crash.SentryInPlace0"		"<clr:238,180,34>Crash: Geschützturm aktiviert!"
```

Comment:
Ich habe die Sentries in der gesamten Übersetzung ["Wächter" genannt](https://github.com/ReactiveDrop/reactivedrop_translations/wiki/Lexicon#weapons) (und nie bemerkt das sie hier gar nicht übersetzt wurden), ich bin der Meinung das sollten wir hier beibehalten.

EDIT: Nein, nein, ich ziehe meine Meinung zurück. Wenn die Soldaten reden können sie eigentlich auch alternative Namen verwenden. Das ist ja nicht wirklich ein Teil der, ich nenne es jetzt mal "der integrierten Spielanleitung".